wp-content/themes/core/pcss/components/button/_outline.pcss

```diff
@@ -0,0 +1,71 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Button: Outline
+ *
+ * ----------------------------------------------------------------------------- */
+
+.c-btn-outline,
+a.c-btn-outline {
+	@mixin button-global;
+	@mixin button-base;
```

Comment:
Let's talk through this tomorrow, I could see a case being made for nixing a `button-base` mixin to give us greater, less complicated ability to pivot styles for any given button while keeping down some of the css.

Reply:
Removed the base mixin and added mixins for `button-solid` and `button-outline`. I thought adding one for `button-text` was overkill.